src/rust/src/hardsubx/decoder.rs

```diff
@@ -0,0 +1,334 @@
+#[cfg(feature = "hardsubx_ocr")]
+use leptonica_sys::*;
+
+// #[cfg(feature = "hardsubx_ocr")]
+// use ffmpeg_sys_next::*;
+
+use std::eprintln;
+use std::format;
+use std::process::exit;
+
+#[cfg(feature = "hardsubx_ocr")]
+// use crate::bindings::{hardsubx_ocr_mode_HARDSUBX_OCRMODE_WORD};
+use crate::bindings::AVFrame;
+use crate::hardsubx::classifier::*;
+use crate::hardsubx::imgops::{rgb_to_hsv, rgb_to_lab};
+use crate::hardsubx::lib_hardsubx_ctx;
+use crate::utils::string_to_c_char;
+use std::convert::TryInto;
+use std::ffi;
+use std::os::raw::c_char;
+use std::ptr::null;
```

Comment:
You should group the imports in all the files. I think there is some setting in rust analyzer that helps with it